python/cudf/cudf/core/column/column.py

```diff
@@ -203,17 +203,19 @@ def __repr__(self):
         )
 
     def to_pandas(
-        self, index: Optional[pd.Index] = None, **kwargs
+        self,
+        index: Optional[pd.Index] = None,
+        nullable: bool = False,
     ) -> pd.Series:
         """Convert object to pandas type.
 
         The default implementation falls back to PyArrow for the conversion.
         """
         # This default implementation does not handle nulls in any meaningful
-        # way, but must consume the parameter to avoid passing it to PyArrow
-        # (which does not recognize it).
-        kwargs.pop("nullable", None)
-        pd_series = self.to_arrow().to_pandas(**kwargs)
+        # way
+        if nullable:
+            raise NotImplementedError(f"{nullable=} is not implemented.")
+        pd_series = self.to_arrow().to_pandas()
```

Comment:
**note:** In theory this removes some ability to pass kwargs into the `to_pandas` call, but I doubt anyone was actually doing that.